blockchain/blocks/src/header.rs

```diff
@@ -202,18 +201,6 @@ impl<'de> Deserialize<'de> for BlockHeader {
     }
 }
 
-impl PartialOrd for BlockHeader {
-    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
-        self.cached_bytes.partial_cmp(&other.cached_bytes)
-    }
-}
-
-impl Ord for BlockHeader {
-    fn cmp(&self, other: &Self) -> Ordering {
-        self.cached_bytes.cmp(&other.cached_bytes)
-    }
-}
-
```

Comment:
@dutterbutter Wasn't there a specific need for this? I remember vaguely it being written into the spec

Reply:
I vaguely remember us needing this at some point as well. Clearly not anymore?

Reply:
I don't recall the specific need for this, I think its fine to be removed.